knative-operator/pkg/apis/operator/v1alpha1/knativekafka_types.go

```diff
@@ -66,6 +66,17 @@ type Broker struct {
 	// brokers will use, but can be overridden on the individual broker object's config map.
 	// +optional
 	BootstrapServers string `json:"bootstrapServers"`
+
+	// NumPartitions is the number of partitions of a Kafka topic. By default, it is set to 10.
+	NumPartitions int `json:"numPartitions"`
+
+	// ReplicationFactor is the replication factor of a Kafka topic. By default, it is set to 1.
+	ReplicationFactor int `json:"replicationFactor"`
```

Comment:
To make sure we are within the right boundaries:
```suggestion
	NumPartitions int32 `json:"numPartitions"`

	// ReplicationFactor is the replication factor of a Kafka topic. By default, it is set to 1.
	ReplicationFactor int16 `json:"replicationFactor"`
```